### PR TITLE
Fix format video

### PR DESCRIPTION
**Pull request by dacox**

### CODE REVIEW

File: pygetty/formatters.py [hunk 1]
```diff
@@ -67,12 +67,8 @@ def format_video(video):
...
-        video['parsed_dimensions'] = {
-            k: int(v)
-            for k, v in re.search(
-                pattern=MASTERY_DIMENSIONS_REGEX,
-                string=video['mastered_to'],
-            ).groupdict().viewitems()
-        }
+        matches = re.search(pattern=MASTERY_DIMENSIONS_REGEX, string=video['mastered_to'])
+        if matches is not None:
+            video['parsed_dimensions'] = {k: int(v) for k, v in matches.groupdict().viewitems()}
```
Comment (cbowal) [inline pygetty/formatters.py:75]: @dacox currently it looks like downstream we assume `parsed_dimensions` is set.  If we don't get a match, should we just throw an exception?  Or look somewhere else?  Or update the downstream to ignore the result?

File: pygetty/formatters.py [hunk 2]
```diff
@@ -67,12 +67,8 @@ def format_video(video):
...
 
     if video.get('mastered_to') is not None:
-        video['parsed_dimensions'] = {
-            k: int(v)
-            for k, v in re.search(
-                pattern=MASTERY_DIMENSIONS_REGEX,
-                string=video['mastered_to'],
-            ).groupdict().viewitems()
-        }
+        matches = re.search(pattern=MASTERY_DIMENSIONS_REGEX, string=video['mastered_to'])
```
Comment (dacox) [inline pygetty/formatters.py:73]: @klardotsh I wonder if the dimensions pattern is less stable than we thought

**Review by klardotsh — APPROVED** [review]

File: pygetty/formatters.py (hunk 2 above)
Comment (klardotsh) [reply pygetty/formatters.py:73]: It _usually_ works - maybe we could log any cases that don't match to get more insight here?

File: pygetty/formatters.py (hunk 1 above)
Comment (klardotsh) [reply pygetty/formatters.py:75]: We'll get an exception either way; the location of the exception is all that changes. Here we'll get a `NoneType has no attribute groupdict` or something to that effect, with this change we'll push it downstream to become a `KeyError` (which we can talk about in that repo - reminder that `pygetty` is publicly accessible :) )

**Pull request merged**